product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Postgres Enterprise Manager 9.8.0 release notes"
+navTitle: Version 9.8.0
+---
+
+Released: 19th Nov 2024
+
+New features, enhancements, bug fixes, and other changes in PEM 9.8.0 include:
+
+| Type         | Description                                                                                                                                               |
+|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enhancement  | Added support for EDB Advanced Server, EDB Postgres Distributed, and PostgreSQL 17.                                                                       |
+| Enhancement  | Added support for RHEL9 ARM64 platform.                                                                                                                   |
```

Comment:
Need to remember to update the Postgres version matrix for this change.

Reply:
Platform compatibility page is updated by website team